src/merkle/path_set.rs

```diff
@@ -229,8 +240,8 @@ impl MerklePathSet {
     }
 
     /// Returns vector of hashes from current to the root
-    fn compute_path_trace(path: &[Word], depth: u32, index: u64) -> (MerklePath, Word) {
-        let mut pos = 2u64.pow(depth) + index;
+    fn compute_path_trace(path: &[Word], node: NodeIndex) -> (MerklePath, Word) {
```

Comment:
nit: should the parameter be named `index` rather than `node`?

Reply:
Done!